cmd/kubeadm/app/util/config/common_test.go

```diff
@@ -373,6 +501,36 @@ func TestMigrateOldConfigFromFile(t *testing.T) {
 			},
 			expectErr: false,
 		},
+		{
+			desc: "v1beta2: component configs are not migrated",
```

Comment:
is it right that v1beta2 components are not migrated? I was expecting everything different thant v1beta3 ti be migrated to the lates version

Reply:
No, the idea of this test case is to verify, that when migrating an old config file, that contains v1beta1 or v1beta2 kubeadm config and component config(s), only the kubeadm config gets migrated and returned as a result (no component configs are expected in the output).

Reply:
So the test name should be "v1beta2: component configs are migrated" (without not)

Reply:
Nope, completely the opposite. Component configs are ignored, v1beta2 is upgraded to v1beta3 and only that is emitted.
Probably if I change it to something like, would that be less confusing?
```
v1beta2->latest: Ignoring component configs
```